src/Sulu/Bundle/MediaBundle/Media/Storage/StorageInterface.php

```diff
@@ -22,29 +22,37 @@
      *
      * @param $tempPath
      * @param $fileName
-     * @param $version
+     * @param $preferredStorageOptions
      *
      * @return mixed
```

Comment:
The interface should define that the method must return (at the moment) the json encoded storage options string.